common/global_cpu_executor.cpp

```diff
@@ -23,10 +23,14 @@
 
 #include "wangle/concurrent/CPUThreadPoolExecutor.h"
 
+DEFINE_int32(global_worker_threads, sysconf(_SC_NPROCESSORS_ONLN),
+             "The number of threads for global CPU executor.");
+
```

Comment:
This won't work reliably when GetThreadsCount() is called before entering main(), such as static variable initialization. Because the gflag could be initialized after that.